multinode/bootstrapping.md

```diff
@@ -0,0 +1,201 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+Scaling up TimescaleDB is achieved by adding data nodes to an access
+node using the [`add_data_node`][add_data_node] and
+[`attach_data_node`][attach_data_node] functions. In order for a data
+node to be available to the access node it is necessary to ensure that
+it is properly setup to work as a data node. This process is referred
+to as *bootstrapping the data node*.
+
+## Connections to the data node
+
+All connections to the data node are made as the current user on the
+access node. Which requires that:
+
+1. The current user exists on both the data node and access node.
+2. The data node is set up to handle logins using either certificates
+   or password files, as described in the [Data Node Authentication
+   Chapter][distdb-auth].
+
+## Adding data nodes to the access node
+
+While an access node is adding a new data node to the distributed
+database it either bootstraps the data node or verifies that
+the data node is properly bootstrapped.
+
+A new data node is added to the access node using the
+[`add_data_node`][add_data_node] command.
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com');
+```
+
+The access node has to ensure that the database on the data node is
+properly configured by either bootstrapping the database or verifying
+that the database exists and is correctly configured. This is controlled with the
+`bootstrap` option, which can be either `true` (indicating that the
+data node should be bootstrapped) or `false` (indicating that the data
+node will only be validated, but no bootstrapping will take place).
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            boostrap => false);
+```
+
+The default value for the `bootstrap` option is `true`, which means
+that the data node is bootstrapped when added.
+
+The steps to boostrap the data node are:
+
+1. Ensure that there is a database on the data node.
```

Comment:
I think it would be better if this list is upfront instead of last in this section (I think this might also be what @k-rus  asked for). It is generally easier to first have a big picture overview of the process and then drill down into the details. Then the reader knows better what to expect.

Reply:
Tried to reformulate it. PTAL.